README.rst

```diff
@@ -54,11 +54,14 @@ To specify location of your datasets pass ``location`` to ``create_engine()``:
 Table names
 ___________
 
-To query tables from non-default projects, use the following format for the table name: ``project.dataset.table``, e.g.:
+To query tables from non-default projects or datasets, use the following format for the SQLAlchemy schema name: ``[project.]dataset``, e.g.:
 
 .. code-block:: python
 
-    sample_table = Table('bigquery-public-data.samples.natality')
+    # If neither dataset nor project are the default
+    sample_table_1 = Table('natality', schema='bigquery-public-data.samples')
+    # If just dataset is not the default
+    sample_table_2 = Table('natality', schema='bigquery-public-data')
```

Comment:
As discussed in https://github.com/mxmzdlv/pybigquery/pull/53, I believe it's more in the spirit of SQLAlchemy and common SQL database notions of 'schema name' to specify the location, rather than encouraging people to specify it in the name (in BigQuery-speak, the project and dataset).

As per previous discussion, the code will continue to accept dataset and project specified in the table name to avoid breaking existing users, and there are regression tests added for that usage.